### PR TITLE
Update release-options-in-office-365.md

### PR DESCRIPTION
**Pull request by msbemba**
Moved the important tag to the top with updates on Teams not available targetted release

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2444

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/admin/manage/release-options-in-office-365.md [hunk 1]
```diff
@@ -34,10 +34,10 @@ description: "Learn how to set up the release option for new product and feature
...
-  
 > [!IMPORTANT]
-> The Microsoft 365 updates described in this article apply to Microsoft 365, SharePoint Online, and Exchange Online. They do not apply to Skype for Business and related services. These release options are targeted, best effort ways to release changes to Microsoft 365 but cannot be guaranteed at all times or for all updates. 
+> The Microsoft 365 updates described in this article apply to Microsoft 365, SharePoint Online, and Exchange Online. These release options are targeted, best effort ways to release changes to Microsoft 365 but cannot be guaranteed at all times or for all updates. They do not apply to Skype for Business, Microsoft teams and related services
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/release-options-in-office-365.md]:
```suggestion
> The Microsoft 365 updates described in this article apply to Microsoft 365, SharePoint Online, and Exchange Online. These release options are targeted, best effort ways to release changes to Microsoft 365 but cannot be guaranteed at all times or for all updates. They do not apply to Skype for Business, Microsoft Teams, and related services.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks.

**Pull request merged**